docs/articles_en/openvino_workflow/gen_ai.md

```diff
@@ -0,0 +1,144 @@
+# Optimize and Deploy Generative AI Models {#gen_ai_guide}
+
+@sphinxdirective
+
+OpenVINO toolkit could be used to accelerate Generative AI use cases as those mostly rely on model inference. When it comes to Generative Models, OpenVINO supports:
+
+* Conversion, optimization and inference for Text, Image and Audio generative models (for example, llama2, MPT, OPT, Stable Diffusion, Stable Diffusion XL, etc.) 
+* Int8 weights compression for text generation models 
+* Reduced storage format (fp16 precision for non-compressed models and int8 for compressed models) 
+* Inference on CPU and GPU platforms, including Intel integrated and discrete ARC GPUs 
+ 
+
+There are two main paths how OpenVINO could be used to run Generative AI use cases:
```

Comment:
I don't know if we want to complicate this document, and I don't know the status of that for LLMs, but for stable diffusion, toch.compile() is also a good option.

Reply:
It is experimental for LLMs still from what I know. But we can improve this in the next version when it is more mature.